.github/workflows/end-to-end-testing-dataset-browser.yml

```diff
@@ -0,0 +1,28 @@
+name: End-to-end testing Dataset Browser
+on:
+  pull_request:
+    branches: ["main"]
+    paths:
+      - 'package.json'
+      - 'apps/dataset-browser/**'
+      - .github/workflows/end-to-end-testing-dataset-browser.yml
+jobs:
+  run-end-to-end-tests:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Waiting for 200 from the Vercel Preview
```

Comment:
Minor:
1. 'Wait' (to be consistent with the other names)?
2. What is the meaning of 200? Wait for 200 milliseconds? Can we make this explicit?